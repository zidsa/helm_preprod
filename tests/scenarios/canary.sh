#!/bin/bash

INPUT_TOKEN=foo \
INPUT_SECRETS='{"secret": "val"}' \
INPUT_CHART=app \
INPUT_NAMESPACE=default \
INPUT_RELEASE=app \
INPUT_VERSION=1234 \
INPUT_TRACK=canary \
INPUT_VALUES='{"replicaCount": 1, "image": {"repository": "nginx", "tag": "latest"}}' \
node ../index.js
