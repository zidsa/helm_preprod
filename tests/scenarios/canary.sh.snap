::debug::not setting deployment status
::debug::param: track = "canary"
::debug::param: release = "app-canary"
::debug::param: appName = "app"
::debug::param: namespace = "default"
::debug::param: chart = "/usr/src/charts/app"
::debug::param: chart_version = ""
::debug::param: values = "{"replicaCount": 1, "image": {"repository": "nginx", "tag": "latest"}}"
::debug::param: dryRun = ""
::debug::param: task = ""
::debug::param: version = "1234"
::debug::param: secrets = "{"secret":"val"}"
::debug::param: valueFiles = "[]"
::debug::param: removeCanary = 
::debug::param: timeout = ""
::debug::param: repository = ""
::debug::param: atomic = "true"
::debug::env: KUBECONFIG="undefined"
::debug::rendering value files [./values.yml] with: {"secrets":{"secret":"val"}}
[command]helm upgrade app-canary /usr/src/charts/app --install --wait --namespace=default --set=app.name=app --set=app.version=1234 --values=./values.yml --set=service.enabled=false --set=ingress.enabled=false --atomic
helm upgrade app-canary /usr/src/charts/app --install --wait --namespace=default --set=app.name=app --set=app.version=1234 --values=./values.yml --set=service.enabled=false --set=ingress.enabled=false --atomic
::debug::not setting deployment status
